acceptance/suites/pre_suite/foss/71_smoke_test_puppetserver.rb

```diff
@@ -0,0 +1,23 @@
+# This step needs to execute as quickly as possible after the
+# install_puppet_server master step located in 70_install_puppet.rb
+#
+# This is because beaker uses `puppet resource` to manage the puppetserver
+# service which has the side-effect of changing ownership and permissions of key
+# paths such as /var/run/puppetlabs and /etc/puppetlabs/puppet/ssl
+#
+# This side-effect masks legitimate issues we need to test, such as "the
+# puppetserver fails to start out of the package"
+
+step "(SERVER-414) Make sure puppet-server can start without puppet resource, "\
+  "apply, or agent affecting the known good state of the SUT in a way that "\
+  "causes the tests to pass with false positive successful results."
+
+variant = master['platform'].to_array.first
+case variant
+  when /^(redhat|el|centos)$/
```

Comment:
The exact approach won't work for other variants, no.  Not ever platform implements `service foo {start,stop,status}`.

Beaker got around this by using `puppet resource service puppetserver ensure=...`, but doing that is the root cause of this issue in the first place.

For each variant we need to figure out the service management framework interface, e.g. launchctl, systemctl, /etc/init.d/foo, svcadm, etc...